### ISSUE TITLE
Wrong fields inferencing

### ISSUE THREAD

**Issue by aborruso** [duckdb/duckdb#6690]
### What happens?

Hi,
I have a CSV in which I have a floating number field (the decimal separator is `,`).

When I try to convert it in parquet I have "Invalid Input Error, because duckdb reads `80000000,00` and it expects a date string.

### To Reproduce

Simply run the below command, using this CSV input https://gist.github.com/01c663ecd4b439c9a663dc2e2c35de4a

```bash
echo "COPY (SELECT * FROM read_csv_auto('d.csv',SEP=',',SAMPLE_SIZE=-1,header=TRUE,decimal_separator=',')) TO 'd.parquet' (FORMAT 'PARQUET', CODEC 'Snappy');" |  duckdb
```

### OS:

Debian 11 x64

### DuckDB Version:

v0.7.1 b00b93f0b1

### DuckDB Client:

cli

### Full Name:

Andrea Borruso

### Affiliation:

onData

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Mytherin** [comment]
Thanks for the report! We currently don't yet infer the `decimal_separator` parameter. We should indeed add support for that.

**Comment by aborruso** [comment]
> Thanks for the report! We currently don't yet infer the `decimal_separator` parameter. We should indeed add support for that.

Thank you @Mytherin, but there is something I don't understand or I probably didn't explain myself well.

In my command I set `decimal_separator=','`, so no inferencing is needed. 
Moreover, why if we have `80000000,00` string, duckdb returns me an error related to `[YYYY-MM-DD ]HH:MM:SS[.MS]`?

I'm referring to the inferencing of this error, not to the one of decimal separator.

Thank you again

**Comment by Mytherin** [comment]
You are correct actually - this is triggering a bug due to the NULL values in the file. I will push a fix.

**Comment by aborruso** [comment]
You are very kind, thank you @Mytherin
